Source/inv_iterators.hpp

```diff
@@ -261,6 +261,37 @@ class BeltPlayerItemsRange {
 	Player *player_;
 };
 
+/**
+ * @brief A range over non-equipped player items in the following order: Inventory, Belt.
+ */
+class InventoryAndBeltPlayerItemsRange {
+public:
+	explicit InventoryAndBeltPlayerItemsRange(Player &player)
+	    : player_(&player)
+	{
+	}
+
+	[[nodiscard]] ItemsContainerListRange::Iterator begin() const // NOLINT(readability-identifier-naming)
+	{
+		return ItemsContainerListRange::Iterator({
+		    InventoryPlayerItemsRange(*player_).begin(),
+		    BeltPlayerItemsRange(*player_).begin(),
+		});
+	}
+
+	[[nodiscard]] ItemsContainerListRange::Iterator end() const // NOLINT(readability-identifier-naming)
+	{
+		auto result = ItemsContainerListRange::Iterator({
+		    InventoryPlayerItemsRange(*player_).end(),
+		    BeltPlayerItemsRange(*player_).end(),
+		});
+		return result;

```

Comment:
Can't we just return the iterator directly like we did in line 276? Would be good if for consistency alone.

Reply:
Done